estate/models/estate_property.py

```diff
@@ -0,0 +1,31 @@
+from odoo import fields, models
+from dateutil.relativedelta import relativedelta
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
+    _description = "My Estate Property"
+
+    name = fields.Char(required = True)
+    postcode = fields.Char()
+    date_availability = fields.Date(copy = False, default = fields.Date.today() + relativedelta(months = 3))
+    expected_price = fields.Float(required = True)
+    selling_price = fields.Float(readonly = True, copy = False)
+    bedrooms = fields.Integer(default = 2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
```

Comment:
Always set the label of the field. even if it's obvious and even if the orm will generate one for you using the field name.
postcode = fields.Char("Postcode") or postcode = fields.Char(string="Postcode")
-> better the first in this case, unless string is not the first param. (typically for relational fields)